lib/app/components/organisms/message_bar.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_form_builder/flutter_form_builder.dart';
+
+class MessageBar extends StatelessWidget {
+  const MessageBar({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Align(
+      alignment: Alignment.bottomCenter,
+      child: Container(
+        padding: const EdgeInsets.only(
+          left: 10,
+          top: 10,
+          right: 10,
+          bottom: 10,
```

Comment:
```suggestion
          bottom: 20,
```
ご対応ありがとうございます。
細かい指摘で申し訳ないですが、iPhone14などのベゼルレスタイプのiPhoneだと
bottomが詰まりすぎてるようなので、20くらいに指定してもらっていいですか？
ご確認よろしくお願いいたします🙇
<img src="https://user-images.githubusercontent.com/25321380/204012468-f96033f7-1012-47b5-92fd-b1611ab1538b.png" width=250>

Reply:
@king-kazu39 
おお、すいません、そんな問題が....
ご指摘の通りbottomを20に修正してiOSシミュレータでも確認したのでご確認いただけますか。